选修课程/计算机视觉/HITCV_EXP/exp2/stitch.py

```
@@ -0,0 +1,203 @@
+        A[2 * i] = [-x, -y, -1, 0, 0, 0, u * x, u * y, u] # 对应于 x 的变换方程
+        A[2 * i + 1] = [0, 0, 0, -x, -y, -1, v * x, v * y, v] # 对应于 y 的变换方程
+        
+    _, _, Vt = np.linalg.svd(A)
+    H = Vt[-1].reshape(3, 3) # 取最小奇异值对应的奇异向量，重塑为3x3矩阵
+    H = H / H[2, 2]
+    
+    return H
+
+def ransac_homography(src_pts, dst_pts, threshold=5.0, max_iter=2000, confidence=0.99):
+    
+    """ 请在此处开始编写你的代码：
```

Ruff (0.15.21)

[warning] Docstring contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF002)